### PR TITLE
fix: fix keyword in schema @common

### PR DESCRIPTION
**Pull request by hetao92**
fix bug #19

### CODE REVIEW

File: app/assets/utils/function.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import _ from 'lodash';
 import { keyWords } from '#assets/config/nebulaQL';
 
 export const handleKeyword = (name: string) => {
-  return keyWords.includes(name) ? '`' + name + '`' : name;
+  return keyWords.includes(name.toLowerCase()) ? '`' + name + '`' : name;
```
Comment (huaxiabuluo) [inline app/assets/utils/function.ts]:
```ts
return keyWords.includes(name.toLowerCase()) ? `\`${name}\`` : name;
```

File: app/assets/utils/gql.ts [hunk 2]
```diff
@@ -198,10 +198,11 @@ export const getAlterGQL = (params: {
     const date = config.fields
       .map(item => {
         const { name, type, value, fixedLength, allowNull, comment } = item;
+        const propertyName = handleKeyword(name);
         if (action === 'DROP') {
-          return name;
+          return propertyName;
         } else {
```
Comment (huaxiabuluo) [inline app/assets/utils/gql.ts]: unnecessary else

File: app/assets/index.tsx [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
 import ReactDom from 'react-dom';
 import { Provider } from 'react-redux';
 import { BrowserRouter as Router } from 'react-router-dom';
 
 import App from './App';
 import { store } from './store';
-
+message.config({
+  maxCount: 1,
+});
```
Comment (jerryliang64) [inline app/assets/index.tsx:11]: Does it need also in explorer？

**Review by jerryliang64 — APPROVED** [review]
LGTM

**Pull request merged**